Description: Add a quote to the computers datfile
Author: Andrea Colangelo <warp10@ubuntu.com>
Bug-Debian: http://bugs.debian.org/617612
Last-Update: 2014-06-27

--- fortune-mod-1.99.1.orig/datfiles/computers
+++ fortune-mod-1.99.1/datfiles/computers
@@ -5550,3 +5550,8 @@ kind of access, but there are no forward
 authors.
     -- Jonathan Corbet in "Who needs /dev/kmem?"
        (http://lwn.net/Articles/147901)
+%
+Step 1: Close AutoCAD - I know this can be difficult for some of you. You could
+also wait for AutoCAD to crash if you want.
+    -- Michael Rotolo
+       (http://themadcadder.blogs.com/my_weblog/2008/12/external-xref-manager-xref-path-saver.html)
