Description: Add Jonathan Corbet quote
Origin: other, http://lwn.net/Articles/147901/
Bug-Debian: http://bugs.debian.org/325060
Last-Update: <2013-05-24>

--- fortune-mod-1.99.1.orig/datfiles/computers
+++ fortune-mod-1.99.1/datfiles/computers
@@ -5545,3 +5545,8 @@ For infrastructure technology, C will be
     -- Dennis Ritchie (1941-2011), creator of the C programming language and of
     UNIX
 %
+It has been suggested that root kits are the largest user community for this
+kind of access, but there are no forward compatibility guarantees for root kit
+authors.
+    -- Jonathan Corbet in "Who needs /dev/kmem?"
+       (http://lwn.net/Articles/147901)
