Description: Fix incorrect version of "The Irish Ballad" 
Origin: other, http://sniff.numachi.com/pages/tiRICTICTN.html
Bug-Debian: http://bugs.debian.org/694306
Forwarded: not-needed
Last-Update: 2013-06-24

--- fortune-mod-1.99.1.orig/datfiles/off/unrotated/songs-poems
+++ fortune-mod-1.99.1/datfiles/off/unrotated/songs-poems
@@ -1325,36 +1325,44 @@ Big, fat, juicy ones,
 Little, skinny, cute ones,
 Watch how they wiggle and they squirm.
 %
-Now of a maid, I'll sing a song,
-Sing, rikkity-tikkity-tin.
-Now of a maid, I'll sing a song,	She didn't like her Uncle Zeke,
-Who didn't keep her family long.	Sing, rikkity-tikkity-tin,
-Not only did she do them wrong,		She didn't like her Uncle Zeke,
-She did every one of them in, them in,	And so she drowned him in the creek.
-She did every one of them in.		The water we had was bad for a week,
-					So we had to make do with gin, with gin,
-She weighted her father down with stones.  We had to make do with gin.
-Sing, rikkity-tikkity-tin.
-She weighted her father down with stones,  Her mother she could never stand,
-And sent him off to Davy Jones.		Sing, rikkity-tikkity-tin.
-All that we ever found were bones,	Her mother she could never stand,
-And occasional pieces of skin, of skin, And so a cyanide soup she planned.
-Occasional pieces of skin.	     Her mother died with the spoon in her hand,
-					And her face in a hideous grin, a grin.
-She set her sister's hair on fire,	Her face in a hideous grin.
-Sing, rikkity-tikkity-tin.
-She set her sister's hair on fire,	One day, when she had nothing to do,
-And as the smoke and flames grew higher,  Sing, rikkity-tikkity-tin.
-She danced around the funeral pyre,	One day, when she had nothing to do,
-Playing the violin, -olin,		She cut her baby brother in two,
-Playing the violin.			And served him up as an Irish stew,
-					And invited the neighbors in, -bors in,
-And when at last the police came by,	Invited the neighbors in.
-Sing, rikkity-tikkity-tin.
-And when, at last, the police came by,	For to do so she would have to lie,
-Her little pranks she did nor deny,	And lying, she knew, was a sin, a sin,
-					Lying she knew was a sin.
-		-- "Rikkity-tikkity-tin"
+About a maid I'll sing a song
+Sing rickety tickety tin
+About a maid I'll sing a song		One morning in a fit of pique
+Who didn't have her family long		Sing rickety tickety tin
+Not only did she do them wrong		One morning in a fit of pique
+She did every one of them in, them in	She drowned her father in the creek
+She did every one of them in.		The water tasted bad for a week
+					And we had to make do with gin, with gin
+Her mother she could never stand	We had to make do with gin	
+Sing rickety tickety tin
+Her mother she could never stand	She weighted her brother down with stones
+And so a cyanide soup she planned	Sing rickety tickety tin
+The mother died with the spoon in her hand	She weighted her brother down with stones
+And her face in a hideous grin, a grin	And sent him off to Davey Jones
+He face in a hideous grin.		All they ever found were some bones
+					And occasional pieces of skin, of skin
+She set her sister's hair on fire	Occasional pieces of skin.
+Sing rickety tickety tin
+She set her sister's hair on fire	One day she had nothing to do
+And as the smoke and flame rose higher	Sing rickety tickety tin
+Danced around the funeral pyre		One day she had nothing to do
+Playing a violin, olin			She cut her baby brother in two
+Playing a violin.			And served him up as an Irish stew
+					And invited the neighbors in, bors in
+And when at last the police came by	Invited the neighbors in.
+Sing rickety tickety tin
+And when at last the police came by	And just one thing before I go
+Her little pranks she did not deny	Sing rickety tickety tin
+To do so she would have had to lie	And just one thing before I go
+And lying she knew was a sin, a sin	There's something I think that you ought to know
+And lying she knew was a sin.		They had no proof, so they let her go
+					And they say that she's tall and thin, and thin
+My tragic tale I won't prolong		They say that she's tall and thin.
+Sing rickety tickety tin
+My tragic tale I won't prolong		You've yourself to blame if it's too long
+I hope you lile my little song		You should never have let me begin, begin
+					You should never have let me begin.
+		-- "The Irish Ballad (Rickety Tickety Tin)"
 %
 O!  If I were a fish
 I'd lay hap'ly on my dish.
