Description: Add a quote to the law datfile.
Author: Andrea Colangelo <warp10@ubuntu.com>
Bug-Debian: http://bugs.debian.org/697252
Last-Update: 2014-06-27

--- fortune-mod-1.99.1.orig/datfiles/law
+++ fortune-mod-1.99.1/datfiles/law
@@ -1285,3 +1285,10 @@ they make a law it's a joke.
     -- Justice Murphy, in concurrance, West Virginia State Board of
        Education v. Barnette 319 U.S. 624 No. 591
 %
+Hereâs the thing about lawyers: they donât write to communicate. Lawyers
+are like programmers, their code doesnât have to be human-readable. It may
+even be deliberately obfuscated. It just has to run on the target system and
+produce the desired outcome. In the case of terms-of-use language, the target
+system is the courts, and the desired outcome is income.
+    -- John Shade, ``Shady Illuminations'', PragPub #43, January 2013
+       http://pragprog.com/magazines/2013-01/shady-illuminations
