Description: Fix typo
Author: Andrea Colangelo <warp10@ubuntu.com>
Bug-Debian: http://bugs.debian.org/683633
Last-Update: 2014-06-25

--- fortune-mod-1.99.1.orig/datfiles/songs-poems
+++ fortune-mod-1.99.1/datfiles/songs-poems
@@ -3783,7 +3783,7 @@ M-O-U-S-E.
 %
 Now let the song begin!  Let us sing together
 Of sun, star, moon and mist, rain and cloudy weather,
-Light on the budding leag, dew on the feather,
+Light on the budding leaf, dew on the feather,
 Wind on the open hill, bells on the heather,
 Reeds by the shady pool, lilies on the water:
 Old Tom Bombadil and the River-daughter!
