Description: Fix typo in datfiles/politics
Author: Alexander Klauer <Graf.Zahl@gmx.net>
Bug-Debian: http://bugs.debian.org/683069
Bug-Debian: http://bugs.debian.org/692022
Last-Update: <2014-06-24>

--- fortune-mod-1.99.1.orig/datfiles/politics
+++ fortune-mod-1.99.1/datfiles/politics
@@ -876,7 +876,7 @@ with income tax policies.
 		-- William F. Buckley
 %
 I would much rather have men ask why I have no statue, than why I have one.
-		-- Marcus Procius Cato
+		-- Marcus Porcius Cato
 %
 I would rather be a serf in a poor man's house and be above ground than
 reign among the dead.
--- fortune-mod-1.99.1.orig/datfiles/men-women
+++ fortune-mod-1.99.1/datfiles/men-women
@@ -702,7 +702,7 @@ When should a man marry?  A young man, n
 %
 For I swore I would stay a year away from her; out and alas!
 but with break of day I went to make supplication.
-		-- Paulus Silentarius, c. 540 A.D.
+		-- Paulus Silentiarius, c. 540 A.D.
 %
 For thirty years a certain man went to spend every evening with Mme. ___.
 When his wife died his friends believed he would marry her, and urged
